### PR TITLE
Add e2e test to check default grafana storage size for statefulset type

### LINKED ISSUE #2615: Monitoring Persistent Volume 'size' field should automatically enter `Gi` for the user
currently, there is some template that auto-fills some of the entries when enabling prometheus persistent storage:
Screen Shot 2021-04-02 at 10 38 29 AM

but no autofill occurs for the grafana persistent storage when enabled:
Screen Shot 2021-04-02 at 10 38 16 AM

I think we should change this to something similar to what we have in the UI for HPA Metrics in dashboard, where the data type (?) is auto filled for the user, and the user only has to input a value (see example below)
Screen Shot 2021-04-02 at 10 50 45 AM

in monitoring v1, there are values in by default for grafana PV:
Screen Shot 2021-04-02 at 10 56 26 AM

So at minimum, I think we should include some default values for grafana in v2

### PR DESCRIPTION
**Pull request by momesgin**
### Summary
Fixes #2615 

### Checklist
- [ ] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [ ] The PR has a Milestone 
- [ ] The PR template has been filled out
- [ ] The PR has been self reviewed 
- [ ] The PR has a reviewer assigned
- [ ] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [ ] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: shell/config/types.js [hunk 1]
```diff
@@ -313,3 +313,5 @@ export const AUTH_TYPE = {
 };
 
 export const LOCAL_CLUSTER = 'local';
+
+export const DEFAULT_GRAFANA_STATEFULSET_STORAGE_SIZE = '10Gi';
```
Comment (izaac) [inline shell/config/types.js:317]: I didn't knew this existed, we should be using it on e2e more often. 👍🏽

**Review by izaac — COMMENTED** [review]
Looks good, I just asked a question in the issue to Caleb

**Comment by momesgin** [comment]
@izaac I made a small change to cover pvc storage type as well, sorry for the last minute change

**Review by izaac — APPROVED** [review]
LGTM :shipit:

**Pull request merged**